indico/modules/rb/models/blockings.py

```diff
@@ -109,7 +109,7 @@ def can_override(self, user, room=None, explicit_only=False, allow_admin=True):
         if not explicit_only:
             if allow_admin and rb_is_admin(user):
                 return True
-            if room and room.can_manage(user):
+            if room and room.can_manage(user, allow_admin=allow_admin):
```

Comment:
Would it make sense to still use the `allow_admin` here?
From what I read here, if `allow_admin` is True and the user is actually a room booking admin, it will early return true. Thus, causing this to have no effect.

Is there any occasion where `allow_admin` is still useful even if the user isn't one?

Reply:
`allow_admin` indicates whether admin permissions should be checked at all. so if it's false, we do not return early above, so we don't want the `can_manage` check to return early either